pronto_quadruped_ros/src/legodo_handler_ros.cpp

```diff
@@ -272,7 +272,7 @@ LegodoHandlerBase::Update* LegodoHandlerBase::computeVelocity(){
                                                 Update::legodo,
                                                 utime_);
   } else {
-    ROS_WARN("[LegodoHandlerBase::computeVelocity] Could not estimate velocity");
+    ROS_WARN_THROTTLE(1, "[LegodoHandlerBase::computeVelocity] Could not estimate velocity");
```

Comment:
returning a `nullptr` is not exceptional, it's a thing that happens normally when there's a flight phase. Instead of throttling I would just print it only if in debug mode